File: Java/프로그래머스/0/120849. 모음 제거/모음 제거.java
class Solution {
    public String solution(String my_string) {
        int n=0;
        int count=0;
        char[] charArray = my_string.toCharArray();
        for(int i=0; i<charArray.length; i++){
            if(charArray[i]!='a' && charArray[i]!='e' &&charArray[i]!='i' && charArray[i]!='o' &&charArray[i]!='u'){
                count++;
            }
        }
        char ch[]= new char[count];
        for(int i=0; i<charArray.length;i++){
            if(charArray[i]!='a' && charArray[i]!='e' &&charArray[i]!='i' && charArray[i]!='o' &&charArray[i]!='u'){
                ch[n]=charArray[i];
                n++;
            }
        }
        String answer = String.valueOf(ch);
        return answer;
    }
}

File: 프로그래머스/0/120849. 모음 제거/모음 제거.py
def solution(my_string):
    answer = ''
    for i in range(len(my_string)):
        if(my_string[i] != 'a' and my_string[i] != 'o' and my_string[i] != 'i' 
           and my_string[i] != 'u' and my_string[i] != 'e'):
            answer += my_string[i];
    return answer